src/components/sbb-card/readme.md

```diff
@@ -0,0 +1,35 @@
+# sbb-card
+
+The sbb-card is a container in which is possible to put other html tag. It provides a slot in which is possible to put the `<sbb-card-badge>`.
+
+## Usage
+
+The example below show how to render the component with `<sbb-card-badge>`:
+
+```html
+<sbb-card>
+    <sbb-card-badge slot="badge" appearance="primary" is-discount></sbb-card-badge>
```

Comment:
- Fix tabulation
- Add some other content beside the badge (otherwise card is meaningless)